tokens/src/commands.rs

```diff
@@ -179,21 +213,25 @@ fn transfer<S: Signer>(
 }
 
 fn distribution_instructions(
-    allocation: &Allocation,
+    allocation: &AllocationV2,
     new_stake_account_address: &Pubkey,
     args: &DistributeTokensArgs,
     lockup_date: Option<DateTime<Utc>>,
     do_create_associated_token_account: bool,
 ) -> Vec<Instruction> {
     if args.spl_token_args.is_some() {
-        return build_spl_token_instructions(allocation, args, do_create_associated_token_account);
+        return build_spl_token_instructions(
+            &allocation.into(),
```

Comment:
```suggestion
            &allocation,
```
After you rebase on https://github.com/solana-labs/solana/pull/33937 and update `build_spl_token_instructions()`

Reply:
Done: https://github.com/solana-labs/solana/pull/33926/commits/a77649e051b57d8beabccebd1c553f6c5ab3686f